packages/assets/src/Assets.ts

```diff
@@ -52,6 +52,11 @@ export interface AssetInitOptions
          * ['avif', 'webp', 'png', 'jpg', 'jpeg', 'webm', 'mp4', 'm4v', 'ogv']
          */
         format?: ArrayOr<string>;
+        /**
+         * If true, don't attempt to detect whether browser has preferred formats available.
+         * May result in increased performance as it skips detection step.
+         */
+        strict?: boolean;
```

Comment:
How about calling this "skipDetection"? "strict" is a little too generic/non-descriptive.